babelon/hp-pt.babelon.tsv

```diff
@@ -547,11 +538,6 @@ en	pt	HP:0012035	rdfs:label	Steatocystoma multiplex	Esteatocistoma múltiplo	OFF
 en	pt	HP:0012063	rdfs:label	Aneurysmal bone cyst	Cisto ósseo aneurismático	OFFICIAL	orcid:0009-0005-0193-5224
 en	pt	HP:0012096	rdfs:label	Intracranial epidermoid cyst	Cisto epidermoide intracraniano	OFFICIAL	orcid:0009-0005-0193-5224
 en	pt	HP:0012097	rdfs:label	Intracranial dermoid cyst	Cisto dermoide intracraniano	OFFICIAL	orcid:0009-0005-0193-5224
-en	pt	HP:0012487	rdfs:label	Cerebellopontine angle arachnoid cyst	Cisto aracnoide no ângulo pontocerebelar	OFFICIAL	orcid:0009-0005-0193-5224
-en	pt	HP:0012488	rdfs:label	Intraventricular arachnoid cyst	Cisto aracnoide intraventricular	OFFICIAL	orcid:0009-0005-0193-5224
-en	pt	HP:0012489	rdfs:label	Suprasellar arachnoid cyst	Cisto aracnoide suprasselar	OFFICIAL	orcid:0009-0005-0193-5224
-en	pt	HP:0012540	rdfs:label	Axillary epidermoid cyst	Cisto epidermoide axilar	OFFICIAL	orcid:0009-0005-0193-5224
```

Comment:
FYI, the orcid for this term has changed - which means you have dropped attribution from orcid:0009-0005-0193-5224 for this term. Same goes for some of the other removals. no problem, just to consider (you can add multiple orcids as provenance)